src/plugins/ui_feature/find_and_replace.ts

```diff
@@ -210,6 +210,7 @@ export class FindAndReplacePlugin extends UIPlugin {
     //modulo of negative value to be able to cycle in both directions with previous and next
     nextIndex = ((nextIndex % matches.length) + matches.length) % matches.length;
     this.selectedMatchIndex = nextIndex;
+    this.selection.getBackToDefault();
```

Comment:
maybe put the same comment than in the clipboad for when we'll come back in 6 months and wonder why the hell it's needed